src/ci/docker/run.sh

```diff
@@ -27,6 +27,10 @@ do
   shift
 done
 
+if [[ $image == dist* ]]; then
```

Comment:
I think this also needs a check that DEPLOY_ALT is not set. (I'm not sure what happens if both are set at the same time.)